src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt

```diff
@@ -327,9 +327,12 @@ class IndexingService @Inject constructor(
     private fun logTypeMessage(objectType: ZoekObjectType, message: String) =
         LOG.info("[$objectType] $message")
 
-    private fun logTypeError(objectType: ZoekObjectType, error: Throwable) =
+    private fun logTypeWarning(objectType: ZoekObjectType, error: Throwable) =
         LOG.log(Level.WARNING, "[$objectType] Error during indexing", error)
 
+    private fun logTypeWarning(objectType: ZoekObjectType, message: String) =
```

Comment:
Same here. I would remove this entire function. I don't see the added value at all.